mesheryctl/pkg/utils/platform.go

```diff
@@ -613,6 +614,12 @@ func Startdockerdaemon(subcommand string) error {
 	}
 
 	log.Info("Attempting to start Docker...")
+	// create docker client using ENV variables
+	cli, err := client.NewClientWithOpts(client.FromEnv, client.WithAPIVersionNegotiation())
+	if err != nil {
+		return err
```

Comment:
Understanding that this is in draft...

...a more helpful message could be offered here.

Reply:
For the error?

Reply:
Yes. What information are we providing to users and what do we expect them to do when this occurs?